pkgs/applications/graphics/ocrmypdf/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib, fetchFromGitHub, python3, unpaper, ghostscript, tesseract, qpdf, glibcLocales }:
+
+with python3.pkgs;
+
+buildPythonApplication rec {
+  version = "5.4.3";
+  name = "OCRmyPDF-${version}";
+
+  src = fetchFromGitHub {
+    owner = "jbarlow83";
+    repo = "OCRmyPDF";
+    rev = "v${version}";
+    sha256 = "0vnn6g69vkqldbx76llmyz8h9ia7mkxcp290mxdsydy4bjjik6zf";
+  };
+
+  postPatch = ''
+    substituteInPlace requirements.txt \
+      --replace "ruffus == 2.6.3" "ruffus" \
+      --replace "Pillow == 4.3.0" "Pillow" \
+      --replace "reportlab == 3.4.0" "reportlab" \
+      --replace "PyPDF2 == 1.26.0" "PyPDF2" \
+      --replace "img2pdf == 0.2.4" "img2pdf" \
+      --replace "cffi == 1.11.2" "cffi"
+    substituteInPlace test_requirements.txt \
+      --replace "pytest >= 3.0" "pytest"
+    substituteInPlace ocrmypdf/lib/compile_leptonica.py \
+      --replace "©" "(c)"
+    export SETUPTOOLS_SCM_PRETEND_VERSION="${version}"
+    export LANG=en_US.UTF-8
+    export LC_ALL=en_US.UTF-8
+  '';
+
+  buildInputs = [ pytest pytest_xdist pytestcov setuptools_scm pytest-helpers-namespace pytestrunner glibcLocales ];
+
+  doCheck = false;
```

Comment:
Skipping tests might be fine but please add a comment why you are skipping them.